addons/medical_blood/stringtable.xml

```diff
@@ -3,6 +3,7 @@
     <Package name="medical_blood">
         <Key ID="STR_ACE_medical_blood_subCategory">
             <English>Bleeding Effect</English>
+            <German>Blutungs Effekt</German>
```

Comment:
In deutsch schreibt man sowas nicht auseinander. Entweder mit zusammen oder mit Bindestrich. Blutungseffekt.